policyengine/tests/economic_impact/winners_and_losers/test_by_wealth_decile/test_by_wealth_decile.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import yaml
+import os
+from policyengine import EconomicImpact
+
+def assert_dict_approx_equal(actual, expected, tolerance=1e-4):
```

Comment:
Can we just bump this up to 5e-2? Seems reasonable to me.

Reply:
done.